### ISSUE TITLE
A number of buttons missing screen-reader labels

### ISSUE THREAD

**Issue by Murcielago7384** [bitwarden/clients#21146]
### Steps To Reproduce

Note, for this bug to be detected, you will need to be using a screen-reader (SR). I use NVDA, which is a free and open source SR from NV Access.
Go to any login entry in your Bitwarden desktop client. One of these issues also exists in secure notes, as I will demonstrate below. You now need to cycle through the various buttons that appear on the screen. This can either be done via repeated tab key presses, or by pressing the NVDA  modifier key (which will either be the 'insert' key or 'caps-lock' key) and space simultaneously, and listening out for a lower pitched beep. This is known as browse mode. You can now press 'B' repeatedly to jump through all the buttons displayed on the page. Pay attention to how NVDA identifies each control as you move past it.

### Expected Result

If buttons are coded correctly with the invisible SR label, NVDA should be  be able to detect the functionality of this button. For example, the SR might announce "save" when the 'save' button is focused.

### Actual Result

For the most part, most of the buttons in the Bitwarden desktop client are correctly labelled. However, there are several that are not (see below), which means there is no way to work out their functionality accept by clicking on them, which could have unintended consequences. This reduces the ease and efficiency of using the desktop client for those relying on SRs.

Unlabelled buttons:
* The copy button next to a login entry's URI field.
* The copy button next to a (hidden) custom field entry. (This applies to both 'logins' and 'secure notes'.)

When editing a login:
* The edit button next to any custom field.
(Once this edit button is selected(:
* In the small dialogue box that appears, the 'save' 'cancel' and 'close' buttons have a screen-reader friendly label, but there is one button which does not.

### Additional Context

As stated at the beginning, you may need to install a screen-reader for this to work. NVDA is a free and open source SR available from NV Access. There may be other buttons in the desktop client which are not labelled, though these were the only ones I was able to find.

### Operating System

Windows

### Operating System Version

11 (Latest as of 9 June 2026).

### Installation method

Direct Download (from bitwarden.com)

### Build Version

2026.5.0

### Issue Tracking Info

- [x] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Murcielago7384** [comment]
This issue should include one of Bitwarden's "Accessibility" labels, but I cannot figure out how to do that. Sorry, new GitHub user here!

**Comment by cbbit** [comment]
Hi there,

This report has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by Murcielago7384** [comment]
Thank you for getting this fixed, @jaasen-livefront  :)
